QOS tag section clarification (#88900)
Azure/azure-content: articles/expressroute/expressroute-qos.md

* You should classify the workloads and mark the right DSCP values. Follow the guidance provided [here](/SkypeForBusiness/manage/network-management/qos/configuring-port-ranges-for-your-skype-clients#configure-quality-of-service-policies-for-clients-running-on-windows-10) on how to set DSCP markings in your network.
* You should configure and support multiple QoS queues within your network. Voice must be a standalone class and receive the EF treatment specified in [RFC 3246](https://www.ietf.org/rfc/rfc3246.txt). 
* You can decide the queuing mechanism, congestion detection policy, and bandwidth allocation per traffic class. But, the DSCP marking for Skype for Business workloads must be preserved. 

> * If you are using DSCP markings not listed above, e.g. AF31 (26), you must rewrite this DSCP value to 0 before sending the packet to Microsoft. Microsoft only sends packets marked with the DSCP value shown in the above table. 

Reviewer: ```suggestion
* If you use DSCP markings that aren't listed in the table, such as AF31 (26), you must rewrite the DSCP value to 0 before you send the packet to Microsoft. Microsoft only sends packets marked with DSCP values shown in the table. 
```
